src/app/store/answer/answer.effects.ts

```diff
@@ -169,7 +167,7 @@ export class AnswerEffects {
   loadDetails = this.actions.pipe(
     ofType(AnswerActionTypes.LOAD_DETAILS),
     withLatestFrom(this.store.select(answer)),
-    filter(([, crtAnswerState]) => !!crtAnswerState.selectedAnswer === false),
+    // filter(([, crtAnswerState]) => !!crtAnswerState.selectedAnswer === false),
```

Comment:
^^^ band-aid fix for answers not being reloaded when switching to another answer. CC @Andrei0872

Reply:
That line is what makes sure that data is not refetched if it's already in the store.

You could easily solve this by adding another prop to that action, say "forceRefresh: boolean" and the filter's callback would also include "|| action.forceRefresh".